client/src/components/leftPanel/LeftPanel.js

```diff
@@ -16,6 +18,13 @@ class LeftPanel extends Component {
 
   render() {
     const {tags} = this.props
+    let parts = [[],[]]
+    tags.forEach((tag) => {
+      const isLate = moment().diff(parseTs(tag.lastTs), 'days') > tag.frequency
+      tag.isLate = isLate
+      parts[isLate ? 0 : 1].push(tag)
+    })
+    const partitionedTags = parts[0].concat(parts[1])
```

Comment:
Uff.. :slightly_smiling_face: 

If you have used sort it would have been more readable. JavaScript's built-in sort is not stable in Edge. Let's add lodash to the project and use its sort which is stable. :slightly_smiling_face:

Reply:
I've decided to use partition, I think it's even more readable.